srcpkgs/duc/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'duc'
+pkgname=duc
+version=1.4.4
+revision=1
+build_style=gnu-configure
+hostmakedepends="pkg-config"
+makedepends="pango-devel cairo-devel ncurses-devel tokyocabinet-devel"
+short_desc="Library and suite of tools for inspecting disk usage"
+maintainer="mobinmob <mobinmob@disroot.org>"
+license="LGPL-3.0-only"
+homepage="https://duc.zevv.nl/"
+distfiles="https://github.com/zevv/"$pkgname"/releases/download/"$version"/"$pkgname"-"$version".tar.gz"
```

Comment:
distfiles="https://github.com/zevv/duc/releases/download/${version}/duc-${version}".tar.gz"

Variables expansion need to be quoted. There is no reason to use $pkgname, as it is always duc.
Please do the same in other template.

Reply:
Fixed in both templates, thank you :)